apps/api-documenter/src/cli/ApiDocumenterCommandLine.ts

```diff
@@ -10,13 +10,15 @@ export class ApiDocumenterCommandLine extends CommandLineParser {
   public constructor() {
     super({
       toolFilename: 'api-documenter',
-      toolDescription: 'Reads *.api.json files produced by api-extractor, '
-        + ' and generates API documentation in various output formats.'
+      toolDescription:
+        'Reads *.api.json files produced by api-extractor, ' +
+        ' and generates API documentation in various output formats.',
     });
     this._populateActions();
   }
 
-  protected onDefineParameters(): void { // override
+  protected onDefineParameters(): void {
+    // override
```

Comment:
Should this be in a `/** ... */` comment?

Reply:
`// override` and `// abstract` are a convention I use to remember how the base class member is defined. Eventually should replace them with the TSDoc equivalents (and an ESLint rule to valide them) but it's out of scope for this PR that already churns 838 files :-)